hardhat.config.ts

```diff
@@ -90,6 +90,10 @@ export default {
     hardhat: {
       blockGasLimit: 12.5e6,
       initialBaseFeePerGas,
+      forking: {
```

Comment:
Why not actually making this into a test since we already have [mainnet forking tests](https://github.com/gnosis/cow-token/blob/main/test/mainnet/addresses.test.ts)? Then we can also test for events.